### DISCUSSION TITLE
Number of days of the event

### DISCUSSION THREAD

**Discussion by teamtacho** [liedekef/events-made-easy#257] [General]
Hi
I'd like to display the number of days of the event in the event list if the event takes longer than one day.
Is there a placeholder "number of days" or can you guide me how to calculate.

**Comment by liedekef** [comment]
using custom php you can calculate the difference between $event['event_start'] and $event['event_end'] , but currently no placeholder exists to help you here.

**Reply by liedekef** [reply]
Correction, the placeholder #_DATETIMEDIFF_START_END{xx} with "xx" being a [php date interval format](https://www.php.net/manual/en/dateinterval.format.php)
This placeholder was not documented, correcting it now

**Reply by liedekef** [reply]
And the next version will also support just #_DATETIMEDIFF_START_END (where leaving out the php date interval format will default to showing the difference in days)

**Comment by teamtacho** [comment]
Great!

**Comment by teamtacho** [comment]
Hi Franky
#_DATETIMEDIFF_START_END{d} doesn't show what it has to. In the event-list only a "d" character is displayed

**Reply by liedekef** [reply]
You need to use "%d", per the php datetimeinterval doc on formatting.

**Comment by teamtacho** [comment]
Ok. Works almost right. The number of the days is one less. php has to be End-Start+1

**Reply by liedekef** [reply]
No it does not. It is the exact difference in days. An event starting/ending on the same day has difference in days = 0. This placeholder is an exact difference between event end and start, formatted as a php interval, no corrections are possible.

**Comment by teamtacho** [comment]
Ok. If this is the definition then it is so.

**Reply by liedekef** [reply]
You can always compare if the end-date (in day format) is different as the start date , and show info based on that. Example:
`[eme_if tag="#_ENDDATE" notvalue="#_STARTDATE"]end date is not start date, so this is an event spanning more than 1 day[/eme_if]`

**Comment by teamtacho** [comment]
I did something similar. I have just some events which takes longer than one day. For such an event I put the number of days in a custom field an display it with the eme_if as you described.
Thanks Franky.
